assets/src/util/sentry.tsx

```diff
@@ -19,23 +19,34 @@ const error = (message: string) => log(message, "error");
  * a real production screen.
  */
 const initSentry = (appString: string) => {
-  const { sentry: sentryDsn, environmentName: env } = getDataset();
+  const {
+    sentry: sentryDsn,
+    environmentName: env,
+    disableSentry,
+  } = getDataset();
+
+  console.log(disableSentry);
+
+  if (!disableSentry && isRealScreen()) {
+    console.log("HERE");
```

Comment:
2 things: remove console.logs and update the comment for this function to mention sentry

Reply:
😬